From bea1ec0e89a647ada98ea076f575cf4917da65b9 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Wed, 12 Jun 2019 19:21:10 +0000
Subject: [PATCH] TRITON-1738 update imgapi base image to
 triton-origin-x86_64-18.4.0 TRITON-1737 imgapi should use node v6 Reviewed
 by: Todd Whiteman <todd.whiteman@joyent.com> Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Todd Whiteman <todd.whiteman@joyent.com>

---
 Makefile     | 22 ++++++++++------------
 deps/eng     |  2 +-
 package.json |  2 +-
 3 files changed, 12 insertions(+), 14 deletions(-)

diff --git a/Makefile b/Makefile
index 0b0d68b..04fa066 100644
--- a/Makefile
+++ b/Makefile
@@ -33,28 +33,26 @@ SMF_MANIFESTS = $(shell ls smf/manifests/*.xml)
 NODEUNIT	:= ./node_modules/.bin/nodeunit
 CLEAN_FILES += ./node_modules
 
-NODE_PREBUILT_VERSION=v4.9.0
+NODE_PREBUILT_VERSION=v6.17.0
 ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_TAG=zone
-	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
-	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
+	NODE_PREBUILT_TAG=zone64
+	NODE_PREBUILT_IMAGE=c2c31b00-1d60-11e9-9a77-ff9f06554b0f
 endif
 
 #
 # Stuff used for buildimage
 #
-# our base image is triton-origin-multiarch-15.4.1
-BASE_IMAGE_UUID		= 04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f
+# our base image is triton-origin-x86_64-18.4.0
+BASE_IMAGE_UUID		= a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME		= imgapi
 BUILDIMAGE_DESC		= SDC IMGAPI
 BUILDIMAGE_PKGSRC	= \
-	dateutils-0.3.1nb1 \
-	haproxy-1.6.2 \
-	openssl-1.0.2o \
+	dateutils-0.4.3nb1 \
+	haproxy-1.8.14 \
 	smtools-20160926 \
-	stud-0.3p53nb5 \
-	the_silver_searcher-0.31.0 \
-	xz-5.2.2 \
+	stud-0.3p53nb7 \
+	the_silver_searcher-2.2.0 \
+	xz-5.2.4 \
 
 AGENTS = amon config registrar
 
diff --git a/deps/eng b/deps/eng
index d25b8fc..ed10150 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit d25b8fc60fb8c649559924870fe3aaf75e7421d5
+Subproject commit ed10150657b1f3ae4f061c751daf365ed16ca8d5
diff --git a/package.json b/package.json
index 138e869..3001d85 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for Triton Data Center",
-  "version": "4.6.2",
+  "version": "4.7.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-- 
2.21.0

